pkg/dns/vips/virtual_outbound_view.go

```diff
@@ -47,9 +51,11 @@ func (vo *VirtualOutboundMeshView) Add(entry HostnameEntry, outbound OutboundEnt
 				return nil
 			}
 			if entry.Type == Host {
-				return fmt.Errorf("autogenerated DNS entry %s:%d from %s conflicts with existing entry %s, this happens when there are two external services with the same 'networking.address', to disable automatic DNS generation in external services set 'networking.disableHostDNSEntry=true' in at least one of these externalService", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin)
+				logger.Info("[WARNING]: autogenerated DNS entry %s:%d from %s conflicts with existing entry %s, this happens when there are two external services with the same 'networking.address', to disable automatic DNS generation in external services set 'networking.disableHostDNSEntry=true' in at least one of these externalService", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin)
```

Comment:
TBH I'd use `logger.Error()` so log collectors correctly identify this as an error. This one definitely requires mesh operator attention since one of the deployed services is not part of the mesh (though it's expected to be)

Reply:
Changed logger to Error, and added info about headless services